input/pagecontent/aucdi.md

```diff
@@ -211,9 +211,9 @@ Column attribute descriptions are as follows:
   </tr>
   <tr>
     <td colspan="2">Date of assertion</td>
+    <td>AU Core MedicationStatement</td>
     <td>TBD</td>
-    <td>TBD</td>
-    <td>This data element is added to AUCDI R1; work is underway to map to AU Core.</td>
+    <td>Decision on supporting this AUCDI element will be addressed as part of the work on AU Core MedicationStatement profile in AU Core R2.</td>
```

Comment:
Replace "Decision on supporting this AUCDI element will be addressed as part of the work on AU Core MedicationStatement profile in AU Core R2." with
"Mapping of this element as part of AU Core MedicationStatement profile is planned for AU Core R2."

Reply:
Done